### DISCUSSION TITLE
tema ODS 2022

### DISCUSSION THREAD

**Discussion by pcmasuzzo** [opendatasicilia/ods2022#3] [Ideas]
usiamo questo spazio per fare un po' di brainstorming sul tema in vista della prossima riunione?

**Comment by gbvitrano** [comment]
@pcmasuzzo posso convertire issue in discussione?

**Comment by pcmasuzzo** [comment]
Assolutamente, scusami che non ci ho pensato!

**Reply by gbvitrano** [reply]
fatto :-)
